README.md

```diff
@@ -22,6 +22,12 @@ Copy the sample `.env` file then open it and set the variables.
 ```bash
 cp sample.env .env
 ```
+
+Build the Dockerfile. 
+```bash
+docker-compose build
```

Comment:
Now that docker-compose is building from the Dockerfile rather than an image this is actually not necessary. i.e. `docker-compose up` should automatically build the container if it hasn't already been built.